src/types/exports.ts

```diff
@@ -60,10 +60,15 @@ interface CommonProps {
   pluginUris?: string[]
 
   // Called if something goes wrong when starting the core:
+  // This will change to `(error: unknown) => void`
```

Comment:
When?

Reply:
This will be a breaking change, so whenever we bump the version number to 2.0.